### ISSUE TITLE
Long end segments make the stops along it unreachable

### ISSUE THREAD

**Issue by Dimitar5555** [Zaczero/osm-relatify#80]
relation/12696014 (NE part)
relation/12696013 (SW part)
relation/12696828 (SW part)

When the end segment is very long, Relatify can't seem to reach it's end where the last stop usually is. It also thinks that the stops along that segment are unreachable.

![image](https://github.com/user-attachments/assets/f1157bc7-d79b-4acf-8f8e-1503142d80d8)

**Comment by mutipg** [comment]
I encountered the same problem. The solution that worked was to split the tram line at the previous stop before the final stop in an external app. Although if Relatify took care of it itself it would be great.

**Comment by Zaczero** [comment]
Yes, this is a current limitation of Relatify. We should fix it one day.
